src/webgpu/shader/execution/expression/call/builtin/textureLoad.spec.ts

```diff
@@ -373,10 +373,11 @@ Parameters:
       .combine('S', ['i32', 'u32'] as const)
   )
   .beforeAllSubcases(t => {
-    const { format } = t.params;
+    const { format, texture_type } = t.params;
     t.skipIfTextureFormatNotSupported(format);
-    t.skipIfTextureLoadNotSupportedForTextureType(t.params.texture_type);
-    t.selectDeviceForTextureFormatOrSkipTestCase(t.params.format);
+    t.skipIfTextureLoadNotSupportedForTextureType(texture_type);
+    t.selectDeviceForRenderableColorFormatOrSkipTestCase(format);
```

Comment:
same here. Why does it matter the format is renderable here?

Reply:
Thanks for catching it. Removed the unused `GPUTextureUsage.RENDER_ATTACHMENT`. Fixed